src/main/webapp/pages/projects/settings/pages/references.html

```diff
@@ -8,14 +8,38 @@
 <head>
     <meta charset="UTF-8" />
     <title th:text="#{ReferenceFile.title}">Title</title>
-    <link rel="stylesheet" href="/dist/css/project-reference-files.bundle.css">
+    <webpacker:css entry="project-reference-files"/>
 </head>
 <body>
 <div id="reference-files-root" layout:fragment="settings-content" >
     <div th:text="#{ReferenceFile.loading}">LOADING ...</div>
 </div>
-<th:block layout:fragment="scripts">
-    <script th:src="@{/dist/js/project-reference-files.bundle.js}"></script>
+<th:block layout:fragment="scripts"  th:inline="javascript">
+    <div th:replace="templates/_file.utils :: upload-progress"></div>
+    <div th:replace="templates/_file.utils :: upload-error"></div>
+    <script type="text/ng-template" id="/delete-modal.html">
+        <div id="delete-modal">
+            <div class="modal-header">
+                <h3 class="modal-title">
+                    <span class="fa fa-trash fa-fw"></span>
+                    <span th:text="#{referenceFile.delete-modal.title}">Delete FIles</span>
+                </h3>
+            </div>
+            <div class="modal-body">
+                <h4 th:text="#{referenceFile.delete-modal.body}"></h4>
+                <ul>
+                    <li>{{dCtrl.file.label}}</li>
+                </ul>
+            </div>
+            <div class="modal-footer">
+                <button class="btn btn-default" th:text="#{form.btn.cancel}" ng-click="dCtrl.close()"></button>
+                <button id="deleteBtn" class="btn btn-primary" th:text="#{form.btn.remove}" ng-click="dCtrl.delete()">
+                    __Remove File
+                </button>
+            </div>
+        </div>
+    </script>
+    <webpacker:js entry="project-reference-files"/>
```

Comment:
What's with the big addition here?

Reply:
Sorry this must have been a bad dev merge.  Updated in 7397da348e55b31fc54126d62bd4b7701ab967f3